src/plugins/options/options.ts

```diff
@@ -1,119 +1,90 @@
 import {bind} from '@exadel/esl/modules/esl-utils/decorators/bind';
-import {CSSClassUtils} from '@exadel/esl';
-import {randUID} from '@exadel/esl/modules/esl-utils/misc/uid';
 
 import {UIPPlugin} from '../../core/registration';
+import {OptionConfig, UIPOption} from './option/option';
 
 /**
  * Custom element to provide controls for changing UIP visual appearance.
  * @extends UIPPlugin
  */
 export class UIPOptions extends UIPPlugin {
   static is = 'uip-options';
+  options: Map<string, UIPOption>;
+  optionsConfig: OptionConfig[] = [
```

Comment:
minor cosmetic, maybe it makes sense to name it  `UIPOptionConfig`, cause line upper options type is `Map<string, UIPOption>` , for me it was a little bit confusing